cmd/gin-repod/repo.go

```diff
@@ -279,13 +278,18 @@ func (s *Server) listPublicRepos(w http.ResponseWriter, r *http.Request) {
 
 }
 
+// varsToRepoID checks if a map contains the entries "user" and "repo" and
+// returns them as a store.RepoId or an error if they are missing.
 func (s *Server) varsToRepoID(vars map[string]string) (store.RepoId, error) {
 	iuser := vars["user"]
 	irepo := vars["repo"]
 
-	//TODO: check name and stuff
+	repoId := store.RepoId{iuser, irepo}
```

Comment:
That is a good first step, but I think we also might wanna check if it is a valid name and a valid repo.